### ISSUE TITLE
Structured headers don't seem to support `<ins>`

### ISSUE THREAD

**Issue by rbuckton** [tc39/ecmarkup#350]
I tried using structured headers in https://github.com/tc30/proposal-explicit-resource-management and this was the result:

![image](https://user-images.githubusercontent.com/3902892/130373667-be1ee34d-8a0b-4581-8f79-51303b2b4e27.png)

The source text is this:

```ecmarkup
    <emu-clause id="sec-environment-record-operations-adddisposableresource-env-v-hint" type="abstract operation">
      <h1>
      <ins>
        AddDisposableResource (
          _env_ : a declarative Environment Record,
          _V_ : an ECMAScript language value,
          _hint_ : either ~sync~ or ~async~,
        )
      </ins>
      </h1>
      <dl class="header"></dl>
      <emu-alg>
        1. If _V_ is *null* or *undefined*, return NormalCompletion(~empty~).
        1. Let _resource_ be ? CreateDisposableResource(_V_, _hint_).
        1. Append _resource_ to _env_.[[ResourceStack]].
        1. Return NormalCompletion(~empty~).
      </emu-alg>
    </emu-clause>
```

Is there another way I am supposed to annotate insertions?

**Comment by js-choi** [comment]
For what it’s worth, this is why the [BigInt Math proposal spec](https://jschoi.org/21/es-bigint-math/) currently uses `<ins>` on whole sections, rather than just headings.

**Comment by rbuckton** [comment]
My workaround was to add an `h1.ins` class, though it highlights the whole section and doesn't show as an inserted section in the TOC.

**Comment by rbuckton** [comment]
Wrapping `<ins>` around the algorithm name highlights only the name, but sets the `aoid` to `<ins>...</ins>` and also adds an `<ins>` to the generated paragraph.

Wrapping a parameter with `<ins>` (to indicate a new parameter) makes the parameter disappear from the clause.

**Comment by rbuckton** [comment]
I tried using `<ins class="block">` around the section as @js-choi suggested, but it also causes the section entry in the TOC to no longer be marked as an insertion.

**Comment by js-choi** [comment]
Ah, yes, unfortunately that is also true; no `<ins>` appears in the table of contents.

**Comment by bakkot** [comment]
Sorry about that! Fixed in v9.2.0, which I've just published. The thing you originally tried should now work, including marking it correctly in the TOC. Let me know if there's any place it does not behave as you expect.
